ariadne/contrib/federation/schema.py

```diff
@@ -10,6 +10,7 @@
     GraphQLUnionType,
 )
 
+from ... import SchemaNameConverter
```

Comment:
Make this import use `schema_names` module and reorder imports.

Reply:
Can do

Reply:
Should be updated now